src/Magnum/ImGuiIntegration/Integration.h

```diff
@@ -57,11 +57,27 @@ Example usage:
 #include "Magnum/ImGuiIntegration/visibility.h" /* defines IMGUI_API */
```

Comment:
Update the docs above to mention the string conversion, please ;)

Also feel free to add yourself to the copyright header in the license block above the file.